common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -121,6 +121,19 @@ private static async Task Run(Type testType, PerfOptions options)
                             await RunTestsAsync(tests, options, title);
                         }
                     }
+                    catch (AggregateException ae)
+                    {
+                        foreach (Exception e in ae.InnerExceptions)
+                        {
+                            Console.WriteLine($"Exception: {e}");
+                        }
+                        throw;
+                    }
+                    catch (Exception e)
+                    {
+                        Console.WriteLine($"Exception: '{e.Message}'{Environment.NewLine}{e.StackTrace}");
```

Comment:
Would the same comment about how to log the exception apply here as well?

Reply:
I changed all 3 instances.